fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/InclusionQuerySegmentAggregator.java

```diff
@@ -233,10 +237,10 @@ private void processRevIncludeParameters(StringBuilder queryString, List<Object>
             queryString.append(QuerySegmentAggregator.WHERE_CLAUSE_ROOT).append(" AND ");
             // P1.PARAMETER_NAME_ID=xx AND 
             queryString.append("P1.PARAMETER_NAME_ID=").append(this.getParameterNameId(includeParm.getSearchParameter())).append(" AND ");
-            // P1.STR_VALUE='Patient/'  ||
-            queryString.append("P1.STR_VALUE='").append(includeParm.getSearchParameterTargetType()).append("/' || ");
-            // (SELECT LR.LOGICAL_ID
-            queryString.append("(SELECT LR.LOGICAL_ID ");
+            // P1.STR_VALUE IN 
+            queryString.append("P1.STR_VALUE IN ");
+            // (SELECT 'Patient/' || LR.LOGICAL_ID
+            queryString.append("(SELECT '").append(includeParm.getSearchParameterTargetType()).append("/' || LR.LOGICAL_ID ");
```

Comment:
does this mean _revinclude never work well before?

Reply:
correct.  seems it was broken since the beginning...we just lacked test coverage to show it.